File: Modul-1/Soal 10/PRAK110-2310817110013-RandyFebrian.py
from math import sqrt

alas, tinggi, miring = 5, 12, round(sqrt((5 * 5) + (12 * 12)))
keliling, luas = alas + tinggi + miring, round((alas * tinggi) / 2)

print("Diketahui:")
print("Alas = {} cm".format(alas))
print("Tinggi = {} cm\n".format(tinggi))
print("Jawab:")
print("Sisi A = {} cm".format(tinggi))
print("Sisi B = {} cm".format(miring))
print("Sisi c = {} cm".format(alas))
print("Keliling = {} cm".format(keliling))
print("Luas = {} cm".format(luas))

File: Modul-1/Soal 10/PRAK110-2310817110013-RandyFebrian.c
#include <stdio.h>
#include <math.h>

int main(){
    int alas = 5, tinggi = 12;
    int miring = sqrt((alas * alas) + (tinggi * tinggi));
    int keliling = alas + miring + tinggi, luas = (alas * tinggi) / 2;

    printf("Diketahui:\n");
    printf("alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    printf("Jawab : \n");
    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %d cm\n", miring);
    printf("Sisi C = %d cm\n", alas);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm\n", luas);
    return 0;
}